src/components/node-list/node-list-row.js

```diff
@@ -55,13 +55,14 @@ const NodeListRow = memo(
     type,
     visibleIcon = VisibleIcon,
     invisibleIcon = InvisibleIcon,
+    rowType,
   }) => {
     const VisibilityIcon = checked ? visibleIcon : invisibleIcon;
     const isButton = onClick && kind !== 'filter';
     const TextButton = isButton ? 'button' : 'div';
 
     return (
-      <Container
+      <div
```

Comment:
Was this intended? Looks this would break the `container` prop functionality here?

Reply:
I've reverted back to using `Container` in [this commit ](https://github.com/quantumblacklabs/kedro-viz/pull/479/commits/16a64e9a93671757b9ff14ba3a9996a4e7b02695)